src/server/index.js

```diff
@@ -14,12 +14,9 @@ const mongoose = require('mongoose');
 
 
 // SSR method importing from ./controller/ssr
-import { renderSSRcomponent } from "./controller/ssr";
+const {renderSSRcomponent} = require("./controller/ssr")
 
-
-import {
-  MONGO_URL
-} from '../../config/index'
```

Comment:
did you check eslint check after doing this ?
cause what I can remember is that I have `formatOnSave` on in my vscode setting so surely this might have come from there!!

Reply:
Hey, could you please push (a commit in the master branch) the .vscode workspace settings folder for this repo so that my editor automatically detects these configurations?

Reply:
Yeah will do it soon !. these should be synced with the project's eslint setting. so I guess its gonna be same only with `formatOnSave`. right ?

Reply:
I did some research and I found out if one wants to sync his/her eslint configuration with prettier, then he/she needs to add `eslint-config-prettier`. See [this](https://prettier.io/docs/en/integrating-with-linters.html).

Right now, my systems's prettier config does not match with yours. I tried turning on the `formatOnSave` feature, applied prettier to the project and I see many file changes (via git diff).

Reply:
ohhh...
Can we add that then ?
Also I would like to have a chat with you , so are you on twitter or any where ?

Reply:
> Can we add that then ?

Definitely, I'll add it as a separate PR 🙂 

> Also I would like to have a chat with you, so are you on twitter or anywhere?

@ypahalajani 🤓

Reply:
your DM is closed 😄 ..